File: ejercicio25 operador/java/ejercicio25.java
import java.util.Scanner;

public class OperacionMatematica {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Ingrese el primer número: ");
        double numero1 = scanner.nextDouble();

        System.out.print("Ingrese el segundo número: ");
        double numero2 = scanner.nextDouble();

        System.out.print("Ingrese el operador (+, -, *, /): ");
        char operador = scanner.next().charAt(0);

        double resultado = 0;
        boolean operadorValido = true;

        switch (operador) {
            case '+':
                resultado = numero1 + numero2;
                break;
            case '-':
                resultado = numero1 - numero2;
                break;
            case '*':
                resultado = numero1 * numero2;
                break;
            case '/':
                if (numero2 != 0) {
                    resultado = numero1 / numero2;
                } else {
                    System.out.println("Error: División por cero no permitida.");
                    operadorValido = false;
                }
                break;
            default:
                System.out.println("Operador inválido.");
                operadorValido = false;
                break;
        }

        if (operadorValido) {
            System.out.println("El resultado es: " + resultado);
        }

        scanner.close();
    }
}

File: Ejercicio_25/ejercicio.25.py
def operacion_matematica():
    numero1 = float(input("Ingrese el primer número: "))
    numero2 = float(input("Ingrese el segundo número: "))
    operador = input("Ingrese el operador (+, -, *, /): ")

    if operador == "+":
        resultado = numero1 + numero2
    elif operador == "-":
        resultado = numero1 - numero2
    elif operador == "*":
        resultado = numero1 * numero2
    elif operador == "/":
        if numero2 != 0:
            resultado = numero1 / numero2
        else:
            print("Error: División por cero no permitida.")
            return
    else:
        print("Operador inválido.")
        return

    print("El resultado es:", resultado)

operacion_matematica()
